src/downloadlist.h

```diff
@@ -83,8 +105,12 @@ class DownloadList : public QAbstractTableModel
   //
   bool lessThanPredicate(const QModelIndex& left, const QModelIndex& right);
 
-public slots:
+private:
+  void buildDownload(Download* download);
+  Download* getDownload(int downloadId);
```

Comment:
Not sure if this is really needed since this is private - You can simply use `getDownloadIndex` followed by a check against `-1` and then `m_downloads[index]`. The second function might be confusing in the future.

If this remain, this need to be const qualified.

Reply:
function has been removed